### PR TITLE
#3: kotlin migration

### PR DESCRIPTION
**Pull request by lusicom**

### CODE REVIEW

**Comment by lusicom** [comment]
@BogatovEvgeniy  please review.
Keep in mind here is a WIP partial migration to Kotlin lang. Validators are still on java. Will be done in a scope of #5.

c.c @pphilipp

**Review by pphilipp — COMMENTED** [review]
Nice work !

File: app/src/main/java/com/example/petscareapp/LoginFragment.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    private val inputErrorHandler = InputErrorHandler()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+
+        (requireView().findViewById<View>(R.id.button_signupForm) as Button).setOnClickListener {
+            view.findNavController().navigate(R.id.action_loginFragment_to_signUpFragment)
+        }
+
+
+        (requireView().findViewById<View>(R.id.button_login) as Button).setOnClickListener {
```
Comment (BogatovEvgeniy) [inline app/src/main/java/com/example/petscareapp/LoginFragment.kt:23]: Why don't you replace it` findViewById<View>` with `findViewById<Button>`

File: app/src/main/res/layout/login_fragment.xml [hunk 2]
```diff
@@ -75,19 +74,17 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="@string/don_t_have_an_account"
-            android:textColor="@color/dark_blue"
-            android:textSize="20dp" />
+            android:textSize="20sp" />
```
Comment (BogatovEvgeniy) [inline app/src/main/res/layout/login_fragment.xml:77]: It's preferable to move it in resources so it could be reused.

**Pull request merged**